buildspecs/benchmarks.yml

```diff
@@ -10,4 +10,4 @@ phases:
       - mvn install -P quick -pl :sdk-benchmarks --am
       - mvn install -pl :bom-internal
       - cd test/sdk-benchmarks
-      - mvn exec:exec
\ No newline at end of file
+      - java -jar target/benchmarks.jar -f 1 -wi 8 -i 8 software.amazon.awssdk.benchmark.enhanced.dynamodb
```

Comment:
intended change?

Reply:
just while I'm testing these changes in codebuild